src/ultimate_ai_agent/core/system_map/__init__.py

```
@@ -0,0 +1,49 @@
+    "SystemMapEdgeKind",
+    "SystemMapEdgeOrigin",
+    "SystemMapFeatureDeclaration",
+    "SystemMapNode",
+    "SystemMapNodeKind",
+    "SystemMapOpportunity",
+    "SystemMapSnapshot",
+    "SystemMapSnapshotStore",
+    "SystemMapTruthStatus",
+    "build_default_system_map_snapshot",
+    "discover_system_map_opportunities",
+]
```

Ruff (0.16.1)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)